static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -1,5 +1,7 @@
 .creative__tracking-pixel {
     position: absolute;
+    width: 1px;
```

Comment:
> It makes sense, it's just that I have a difficult time reconciling this with the SoC between HTML and CSS.

Separation of concerns does not mean separation of technologies. If an element 'needs' styling to do its job, all extracting CSS does is split a single concern between two pieces of code that have tight coupling but a loose contract. If the style hook breaks - because a class gets renamed, or something with high specificity tramples your element - there is no way of knowing this before runtime.

> we're so used to switch back and forth between the two when developing/debugging anyway, it's literally why CSS was invented.

But not with the benefit of a crystal ball, and we can see many trends against this approach at work already. Consider how React templates pair markup and the scripts that manipulate them. Or how Vue.js 'vuefiles' embed style snippets alongside the templates they're paired to. In both cases, we marry markup with the things that give that markup its behaviour.